resources/rds-instances.go

```diff
@@ -88,6 +88,7 @@ func (i *RDSInstance) Properties() types.Properties {
 	properties.Set("EngineVersion", i.instance.EngineVersion)
 	properties.Set("MultiAZ", i.instance.MultiAZ)
 	properties.Set("PubliclyAccessible", i.instance.PubliclyAccessible)
+	properties.Set("Tags", i.tags)
```

Comment:
Please add tags like it is done [here](https://github.com/rebuy-de/aws-nuke/blob/master/resources/rds-dbclusterparametergroups.go#L81:L81)

Reply:
Hey, Thanks. Have fixed it.